example/extensions/lib_custom_op/README.md

```diff
@@ -0,0 +1,118 @@
+CustomOp Example and Tutorial
+====
```

Comment:
Sometimes one of the transpilers will complain that this is too short. Recommend making it longer to match the title.

Reply:
Do you mean make the === longer to match the length of the title?

Reply:
yes